tests/store/test_rest_store.py

```diff
@@ -62,6 +68,64 @@ def test_response_with_unknown_fields(self, request):
         assert len(experiments) == 1
         assert experiments[0].name == 'My experiment'
 
+    def _verify_requests(self, http_request, host_creds, endpoint, method, json_body):
+        http_request.assert_called_with(host_creds=host_creds,
+                                        endpoint="/api/2.0/preview/mlflow/%s" % endpoint,
+                                        method=method,
+                                        json=json.loads(json_body))
+
+    @mock.patch('requests.request')
+    def test_requestor(self, request):
+        response = mock.MagicMock
+        response.status_code = 200
+        response.text = '{}'
```

Comment:
It would be nice to also test the API endpoints that don't have an empty response, but I don't think it's critical to do so in this PR.

Reply:
yes -- we can add that. but i wanted to put in some test at least and we always add more.